tdrs-backend/tdpservice/users/models.py

```diff
@@ -38,10 +39,14 @@ def __str__(self):
         """Return the username as the string representation of the object."""
         return self.username
 
-    @property
-    def is_admin(self):
-        """Check if the user is an admin."""
-        return (
-            self.is_superuser
-            or Group.objects.get(name="OFA Admin") in self.groups.all()
-        )
+    def is_in_group(self, group_name: str) -> bool:
```

Comment:
This was moved from `tdpservice/users/permissions.py` - having this as a model method allows us to define additional properties for users as shown below.